course_discovery/apps/api/tests/test_serializers.py

```diff
@@ -250,6 +250,19 @@ def test_include_deleted_programs(self):
         )
         self.assertEqual(serializer.data, self.get_expected_data(self.course, self.request))
 
+    def test_filter_courses_on_edx_org_short_name(self):
+        """
+        Verify courses filtering on edX organization.
+        """
+        edx_org_short_name = 'edX'
+        # Course of edx organization
+        course_2 = CourseFactory(partner=self.partner, authoring_organizations__key=edx_org_short_name)
```

Comment:
Instead of adding a code comment `# Course of edx organization` we can update or variable name `course_2` to `edx_org_course`.

Rule: Name variables/method in such a way that we don't need to add code comments. Since the code can be updated in future and its related tests can be fixes too but there will be no way to validate that the code comments are also updated.

Reply:
Done